packages/roosterjs-content-model-core/lib/corePlugin/copyPaste/CopyPastePlugin.ts

```diff
@@ -154,7 +160,12 @@ class CopyPastePlugin implements PluginWithState<CopyPastePluginState> {
                     isCut,
                 }).range;
 
-                if (newRange) {
+                if (this.customCopyCutEnabled && isClipboardEvent(event)) {
+                    event.preventDefault();
+                    const text = contentModelToText(pasteModel);
+                    event.clipboardData?.setData('text/html', tempDiv.innerHTML);
+                    event.clipboardData?.setData('text/plain', text);
```

Comment:
What if right click on inline image and choose "copy image"?

Reply:
Copy Image does not trigger the event the ClipboardEvent to the editor. So it will work normally, based on my testing

But if selecting the image and copying it with the context menu item, will just add the image to the HTML element.

Reply:
ok